release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -101,16 +101,47 @@ With this release, the installation program uses Cluster API instead of Terrafor
 [id="ocp-4-17-web-console_{context}"]
 === Web console
 
+[id="ocp-4-17-ocp-lightspeed_{context}"]
+==== {ols} is available in the web console
+
+Starting with {product-title} 4.16, {ols} is available for use in the web console.
```

Comment:
Shall we add some new features about OLS here or in `Administrator perspective` section? 
For example [PR](https://github.com/openshift/console/pull/14030) addes Lightspeed hover button and user preference settings to show/hide lightspeed button